projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.html

```diff
@@ -0,0 +1,119 @@
+<ng-container *ngIf="messageEvents$ | async as messageEvents">
+  <div class="container">
+    <div class="cx-avatar-line"></div>
+    <div
+      class="cx-messages"
+      id="cx-messages"
+      (keydown.arrowdown)="focusNextChild($event)"
+      (keydown.arrowup)="focusPreviousChild($event)"
+      [attr.aria-label]="'chatMessaging.messages' | cxTranslate"
+    >
+      <ng-container *ngFor="let message of messageEvents; let i = index">
```

Comment:
if this is still an array of events, I think it's better to call it `messageEvent of messageEvents`

Reply:
I kept the name `message` so that it can be more readable. what do you think?

Reply:
yes you can keep it, we can print the event and see the text inside